html-css/Angel

AngelHenriettaAboah opened the pull request:
Here are files containing the make-up of my personal-portfolio /Developer Cv website I created using Visual Studio Code. It was a challenging yet exciting project since it is the first time I was ever able to create a website from scratch. While it might not be at its best, I am happy to have successfully finish it. cheers to more future projects and the strength to finish well......

AngelHenriettaAboah committed:
Add files via upload

Here are files containing the make-up of my personal-portfolio /Developer Cv website I created using Visual Studio Code. It was a challenging yet exciting project
since it is the first time I was ever able to create a website from scratch. While it might not be at its best, I am happy to have successfully finish it.
cheers to more future projects and the strength to finish well......

MahmoudYounes commented:
nitty gritty: place connection links in a footer at the bottom of the page?

MahmoudYounes commented on 1_html-css/index.html:
```diff
@@ -0,0 +1,97 @@
+            <div class="navbar">
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="about.html">About</a></li>
```
The file is About.html not about.html

MahmoudYounes reviewed (approved):
discussed design issues online. left one comment about a code error in index.html.
overall very nice job!

AngelHenriettaAboah committed:
Changes to my Portfolio as requested-git h/wk 2

AngelHenriettaAboah committed:
Changes to my html-css folder as requested.

AngelHenriettaAboah committed:
Update about.css

Changes made here was regarding code error by change the link title from About.css to about.css.

AngelHenriettaAboah committed:
Update and rename About.html to about.html

Updated and rename About.html to about.html with changes

AngelHenriettaAboah committed:
Rename About.css to about.css

Rename About.css to about.css
